### ISSUE TITLE
Make file-line-error more talkative

### ISSUE THREAD

**Issue by moewew** [MiKTeX/miktex#181]
Currently the option `--file-line-errors` is not documented in https://docs.miktex.org/manual/pdftex.html, but it seems to do the same as `--c-style-errors`. The log output produced with that option has a serious deficiency: it does not actually show all of the error message and it does not open the `?` prompt.

Consider the following example document which produces an undefined control sequence error about `\totallyundefinedcommand`

```latex
\documentclass{article}

\begin{document}
Lorem ipsum

\totallyundefinedcommand
\end{document}
```

The CLI output produced by MikTeX's `pdflatex` when run as `pdflatex --file-line-error test` is
```
This is pdfTeX, Version 3.14159265-2.6-1.40.19 (MiKTeX 2.9.6745 64-bit)
entering extended mode
(test.tex
LaTeX2e <2018-04-01> patch level 5
("C:\Program Files\MiKTeX 2.9\tex\latex\base\article.cls"
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
("C:\Program Files\MiKTeX 2.9\tex\latex\base\size10.clo")) (test.aux)
test.tex:6: Undefined control sequence
[1{C:/ProgramData/MiKTeX/2.9/pdftex/config/pdftex.map}] (test.aux) )
(see the transcript file for additional information)<C:/Program Files/MiKTeX 2.
9/fonts/type1/public/amsfonts/cm/cmr10.pfb>
Output written on test.pdf (1 page, 12651 bytes).
Transcript written on test.log.
``` 

The `.log` file is longer, but the relevant section still only reads
```
test.tex:6: Undefined control sequence
The control sequence at the end of the top line
of your error message was never \def'ed. If you have
misspelled it (e.g., `\hobx'), type `I' and the correct
spelling (e.g., `I\hbox'). Otherwise just continue,
and I'll forget about whatever was undefined.

[1

```

Compare this to the output from `pdflatex test`
```
This is pdfTeX, Version 3.14159265-2.6-1.40.19 (MiKTeX 2.9.6745 64-bit)
entering extended mode
(test.tex
LaTeX2e <2018-04-01> patch level 5
("C:\Program Files\MiKTeX 2.9\tex\latex\base\article.cls"
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
("C:\Program Files\MiKTeX 2.9\tex\latex\base\size10.clo")) (test.aux)
! Undefined control sequence.
l.6 \totallyundefinedcommand
                            
? [1{C:/ProgramData/MiKTeX/2.9/pdftex/config/pdftex.map}] (test.aux) )<C:/Program
 Files/MiKTeX 2.9/fonts/type1/public/amsfonts/cm/cmr10.pfb>
Output written on test.pdf (1 page, 12651 bytes).
Transcript written on test.log.
```
which clearly shows the problematic control sequence and allows to enter the help prompt.

TeX live's `--file-line-error` shows a more helpful message.
`pdflatex --file-line-error test` on TeX live 2018 gives
```
This is pdfTeX, Version 3.14159265-2.6-1.40.19 (TeX Live 2018) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./test.tex
LaTeX2e <2018-04-01> patch level 5
(/usr/local/texlive/2018/texmf-dist/tex/latex/base/article.cls
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
(/usr/local/texlive/2018/texmf-dist/tex/latex/base/size10.clo)) (./test.aux)
./test.tex:6: Undefined control sequence.
l.6 \totallyundefinedcommand
                            
? [1{/usr/local/texlive/2018/texmf-var/fonts/map/pdftex/updmap/pdftex.map}]
(./test.aux) )</usr/local/texlive/2018/texmf-dist/fonts/type1/public/amsfonts/c
m/cmr10.pfb>
Output written on test.pdf (1 page, 12194 bytes).
Transcript written on test.log.
```
The message there actually shows the undefined control sequence and opens the `?` prompt that one could use to find out more about the error.

So my feature request is to implement `--file-line-error` in a way that matches the behaviour of TeX live's `--file-line-error`.
Turn
```
! Undefined control sequence.
l.6 \totallyundefinedcommand
```
into
```
./test.tex:6: Undefined control sequence.
l.6 \totallyundefinedcommand
```

### Prior similar reports
I found https://sourceforge.net/p/miktex/bugs/2310/

### See also
https://tex.stackexchange.com/q/485830/35864

**Comment by kbauer** [comment]
Please note that this issue makes debugging large multi-file documents problematic. When invoking the latex commands manually, it switching between use and omission of `--file-line-error` is an inconvenience, but it is a much more significant hindrance for built in log-parsing in editors. 

See related issue https://github.com/texstudio-org/texstudio/issues/325.

**Comment by edocevoli** [comment]
This will be fixed with [milestone 2.9.7050](https://github.com/MiKTeX/miktex/milestone/15).
